src/modules/account/components/account-deposit/account-deposit.jsx

```diff
@@ -13,6 +13,13 @@ export default class AccountDeposit extends Component {
     address: PropTypes.string.isRequired
   }
 
+  static shapeShiftOnClick(e) {
+    e.preventDefault()
```

Comment:
` e.preventDefault()` and `return false` are more or less redundant.

Reply:
Oops, these were in the original JS provided by ShapeShift, but since I changed the anchor tags to buttons, I suppose neither is needed now. Will remove.